internal/utils/kruize/kruize_api.go

```diff
@@ -196,3 +196,32 @@ func Is_valid_recommendation(recommendation kruizePayload.Recommendation, experi
 		return false
 	}
 }
+
+func Delete_experiment_from_kruize(experiment_name string) {
+	url := cfg.KruizeUrl + "/createExperiment"
+	data := []map[string]string{
+		{"experiment_name": experiment_name},
+	}
+	payload, err := json.Marshal(data)
+	if err != nil {
+		log.Error("unable to marshal payload to json: ", err)
+		return
+	}
```

Comment:
Is this needed? considering the `data` and `experiment_name` are statically defined?
and if we need this, then we should also increase counter and message that we failed to delete exp because we have read the kafka message and in next iteration we will delete next one?,
~~~
kruizeAPIException.WithLabelValues("/deleteExperiment").Inc()
log.Error("error Occured while deleting experiment: ", err)
~~~

Reply:
Agree, It is not needed. I will remove it.